index.bs

```diff
@@ -4586,15 +4575,19 @@ The [=remote end steps=] with |command parameters| are:
 1. Let |wait condition| be the value of the <code>wait</code> field of |command
    parameters| if present, or "<code>none</code>" otherwise.
 
-1. Let |document| be |navigable|'s [=active document=].
+1. [=Reload=] |navigable| with userInvolvement "<code>browser UI</code>".
```

Comment:
so actually using the reload seems to be more complicated since it does not seem to generate a navigation ID.